src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/blades/customerSegments-detail.js

```diff
@@ -1,16 +1,47 @@
 angular.module('virtoCommerce.demoSolutionFeaturesModule')
     .controller('virtoCommerce.demoSolutionFeaturesModule.customerSegmentsDetailController',
-        ['$scope', function ($scope) {
+        ['$scope', 'platformWebApp.bladeNavigationService', function ($scope, bladeNavigationService) {
             const blade = $scope.blade;
+            blade.headIcon = 'fa-pie-chart';
             var formScope;
 
             blade.currentEntity = {};
 
             blade.isLoading = false;
 
+            blade.activeBladeId = null;
+
             $scope.canSave = () => {
                 return formScope && formScope.$valid;
             };
 
             $scope.setForm = (form) => { formScope = form };
+
+            $scope.mainParameters = function () {
+                const parametersBlade = {
+                    id: "mainParameters",
+                    title: "demoSolutionFeaturesModule.blades.customer-segments-parameters.title",
+                    subtitle: 'demoSolutionFeaturesModule.blades.customer-segments-parameters.subtitle',
+                    controller: 'virtoCommerce.demoSolutionFeaturesModule.customerSegmentsParametersController',
+                    template: 'Modules/$(virtoCommerce.DemoSolutionFeaturesModule)/Scripts/blades/mainParameters.tpl.html',
+                    originalEntity: blade.currentEntity,
+                    onSelected: function (entity) {
+                        blade.currentEntity = entity;
+                    }
+                };
+                blade.activeBladeId = parametersBlade.id;
+                bladeNavigationService.showBlade(parametersBlade, blade);
+            };
+
+            $scope.$watch('blade.currentEntity', (data) => {
+                if (data) {
+                    $scope.totalPropertiesCount = 3;
+                    $scope.filledPropertiesCount = 0;
+
+                    $scope.filledPropertiesCount += blade.currentEntity.startDate ? 1 : 0;
```

Comment:
Add IsActive field

Reply:
@pushnitsa  By default Is Active is false and not nullable. When we should treat it as filled and when not? :)

Reply:
@ilyawzrd Create segment with all filled properties and IsActive = true, then change it to false (disable segment). Now 3 of 4 properties filled

Reply:
It is come to filled when we opened main parameters blade and click "Ok" button.